src/jsMain/kotlin/baaahs/JsMain.kt

```diff
@@ -14,17 +13,7 @@ import kotlin.browser.window
 
 fun main(args: Array<String>) {
     @Suppress("ConstantConditionIf", "SimplifyBooleanWithConstants")
-    if (1 + 1 == 3) {
-        // Protection from dead code elimination, these aren't actually called here:
-
-        fun <T> nuffin(): T {
-            @Suppress("UNCHECKED_CAST")
-            return null as T
-        }
-        GadgetDisplay(nuffin()) {}
-
-        GlslPreview(nuffin(), nuffin(), null)
-    }
+    if (1 + 1 == 3) dce()
```

Comment:
This still trips me up when I see this line 😄

Reply:
Yeah, it's ugly and I'm certain there's a better way to do this, I'm just [too lazy](https://kotlinlang.org/docs/reference/javascript-dce.html#excluding-declarations-from-dce) at the moment to find it.